src/App.js

```diff
@@ -21,6 +21,7 @@ function App() {
   const [choiceTwo, setChoiceTwo] = useState(null);
   const [disabled, setDisabled] = useState(false);
   const [highScore, setHighScore] = useState(0);
+  const [runtime, setRuntime] = useState(0);
```

Comment:
I think elapsedTime and setTime are setter that you can use.

Reply:
Ok, will work on this